plugins/uberdev/lib/rate-cap-audit.sh

```
@@ -9,11 +9,30 @@ set -eu
 
 uberdev_rate_cap_audit() {
   local WAVE_YAML="$1" CAP="$2" OUT_YAML="$3"
-  python3 - "$WAVE_YAML" "$CAP" "$OUT_YAML" <<'PY'
-import sys, yaml, hashlib, re
+  # Absolute dir of this lib so the embedded python can import the shared host normalizer
+  # (normalize_host.py — the same single source of truth the runtime wrapper uses).
+  local LIBDIR="$(CDPATH= cd -- "$(dirname -- "${BASH_SOURCE[0]}")" 2>/dev/null && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)